src/Analyzers/CSharp/CodeFixes/InlineDeclaration/CSharpInlineDeclarationCodeFixProvider.cs

```diff
@@ -106,7 +106,6 @@ public override Task RegisterCodeFixesAsync(CodeFixContext context)
             var declaratorLocation = diagnostic.AdditionalLocations[0];
             var identifierLocation = diagnostic.AdditionalLocations[1];
             var invocationOrCreationLocation = diagnostic.AdditionalLocations[2];
-            var outArgumentContainingStatementLocation = diagnostic.AdditionalLocations[3];
```

Comment:
shoudl probably fix the analzer to not provide this either.

Reply:
@CyrusNajmabadi addressed in https://github.com/dotnet/roslyn/pull/44199/commits/d256ecd0d4a162bbc78166c3d9a012482d17e7fc